File: Python/Q_44.py
size=int(input("Enter the size of array:"))
array=[]
for i in range(size):
    n=int(input("Enter element:"))
    array.append(n)
Largest_element= max(array)
print(f"Array: {array}")
print(f"The Largest element in the array: {Largest_element}")

File: javascript/q44.js
function findLargest(arr) {
  if (arr.length === 0) {
    return "Array is empty!";
  }

  let largest = arr[0];

  for (let i = 1; i < arr.length; i++) {
    if (arr[i] > largest) {
      largest = arr[i];
    }
  }

  return largest;
}

let input = prompt("Enter numbers separated by spaces:");
let numbers = input.split(" ").map(Number);

console.log("Largest element:", findLargest(numbers));
